#goal definition
bool use_defaults
bool delete_prev_traj
float32 x
float32 y
float32 z
float32 direction_x
float32 direction_y
float32 direction_z
float32 search_distance
---

#result definition
float32 final_x
float32 final_y
float32 final_z

---

#feedback
float32 current_x
float32 current_y
float32 current_z
